app/templates/releases/lts.hbs

```diff
@@ -62,17 +65,19 @@
 
     <tbody>
       {{#each this.currentlySupportedLTS as |release|}}
-        <tr>
-          <td>{{release.version}}</td>
-          <td>{{format-date-time release.promotionDate "MMMM D, YYYY"}}</td>
-          <td>{{format-date-time (add-weeks release.promotionDate 36) "MMMM D, YYYY"}}</td>
-          <td>{{format-date-time (add-weeks release.promotionDate 54) "MMMM D, YYYY"}}</td>
-        </tr>
+      <tr>
+        <td>{{release.version}}</td>
+        <td>{{format-date-time release.promotionDate "MMMM D, YYYY"}}</td>
+        <td>{{format-date-time (add-weeks release.promotionDate 36) "MMMM D, YYYY"}}</td>
+        <td>{{format-date-time (add-weeks release.promotionDate 54) "MMMM D, YYYY"}}</td>
+      </tr>
       {{/each}}
     </tbody>
   </table>
 </div>
 
 <p>
-  These LTS versions are no longer maintained: 3.12 (Oct 2020), 3.8 (Aug 2019), 3.4 (Jan 2019), 2.18 (Feb 2018), 2.16 (Feb 2018), 2.12 (Apr 2017), 2.8 (Nov 2016), and 2.4 (Apr 2016). Their last minor release dates are shown in parentheses.
-</p>
+  These LTS versions are no longer maintained: 3.16 (May 2020), 3.12 (Oct 2020), 3.8 (Aug 2019), 3.4 (Jan 2019), 2.18
+  (Feb 2018), 2.16 (Feb 2018), 2.12 (Apr 2017), 2.8 (Nov 2016), and 2.4 (Apr 2016). Their last minor release dates are
+  shown in parentheses.
+</p>
```

Comment:
In a separate pull request (or in this one), what do you think about creating a table with 2 columns (with column headers `LTS version` and `Last minor release date`)? As the list of unsupported LTSes grows, I expect the paragraph to be more difficult to read:

```suggestion
  The following LTS versions are no longer maintained. The table shows their last minor release dates:
</p>

<table class="lts-table">
  ...
</table>
```

Another option is to have 3 columns: `LTS version`, `Last minor release version`, `Last minor release date`. From https://libraries.io/npm/ember-source/versions

| LTS version | Last minor release version | Last minor release date |
|:--:|:--:|:--:|
| 3.16 | 3.16.10 | August 2020 |
| 3.12 | 3.12.4 | May 2020 |
...

Reply:
I was searching through github releases, but I see the best way to see last minor releases is using the source link you provided, thanks for catching that. I'll try adding a 3 column table to this PR.